src/PatchKit.Network/HttpPostRequest.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace PatchKit.Network
+{
+    /// <summary>
+    /// Describes HTTP POST request.
+    /// </summary>
+    public class HttpPostRequest : BaseHttpRequest
+    {
+        /// <summary>
+        /// Request query to be sent in body.
+        /// </summary>
+        public string Query { get; set; }
```

Comment:
I think that `Query` isn't the right name for it - usually query of url is the text preceded by `?` character. `Body` would be much better here.